mcs/class/Mono.Profiler.Log/Mono.Profiler.Log.dll.sources

```diff
@@ -12,3 +13,6 @@ Mono.Profiler.Log/LogProfiler.cs
 Mono.Profiler.Log/LogReader.cs
 Mono.Profiler.Log/LogStream.cs
 Mono.Profiler.Log/LogStreamHeader.cs
+Mono.Profiler.Aot/ProfileData.cs
+Mono.Profiler.Aot/ProfileReader.cs
+Mono.Profiler.Aot/Records.cs
```

Comment:
Could we rename the assembly to Mono.Profiler.dll or would it be considered an API break? (now that it is going to contain also Aot profiler API)

Reply:
Yes, it would be. Considering how special this assembly is I am not sure it's working taking the general Mono.Profiler for it